### PR TITLE
Repository deprecation

### PR DESCRIPTION
**Pull request by mfs12**

### CODE REVIEW

File: godsfapi/v3/cmd/examples/main.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// Deprecated: This package was deprected, please visit https://github.com/Duet3D/dsf-go.
```
Comment (wilriker) [inline godsfapi/v3/cmd/examples/main.go:1]: If you want to make it extra nice you could add the full path to the according version in the target URL, i.e. append `/v3` for `v3` and `/v2` for `v2` though that is technically not necessary.

**Review by wilriker — APPROVED** [review]
Added a comment to think about but not a blocker, so I give approval to your changes.

File: godsfapi/v3/cmd/examples/main.go (hunk 1 above)
Comment (mfs12) [reply godsfapi/v3/cmd/examples/main.go:1]: It is intended, it is referring to the github repository and not the go package.

Comment (wilriker) [reply godsfapi/v3/cmd/examples/main.go:1]: Yes, but you point from the existing `/v3` package back to `v1` this way. Just saying.

Comment (mfs12) [reply godsfapi/v3/cmd/examples/main.go:1]: not anymore. packages was moved into a subdirectory.

**Pull request merged**